src/jsvar.c

```diff
@@ -46,7 +46,7 @@ unsigned int jsVarsSize = 0;
 unsigned int jsVarsSize = 0;
 JsVar *jsVars = NULL;
 #else
-JsVar jsVars[JSVAR_CACHE_SIZE];
+JsVar jsVars[JSVAR_CACHE_SIZE] __attribute__((aligned(4)));
```

Comment:
Does this align every array element, or just the start of the array? If it's the first then it's going to make compact_vars on every board useless.

Reply:
just the array (99%sure) you would need attribute the JsVar typedef itself to apply for each element, thin I tested this, not sure now

Reply:
checked with nrf52 build

```
200030b6 g     O .bss   00008039 .hidden jsVars
```
vs
```
200030b8 g     O .bss   00008039 .hidden jsVars
```

size still same 2039, not even divisible by 2